src/OrchardCore.Modules/OrchardCore.Lists/Helpers/ListQueryHelpers.cs

```diff
@@ -19,10 +19,11 @@ internal static Task<int> QueryListItemsCountAsync(ISession session, string list
                     .CountAsync();
         }
 
-        internal static Task<IEnumerable<ContentItem>> QueryListItemsAsync(ISession session, string listContentItemId, Expression<Func<ContentItemIndex, bool>> itemPredicate = null)
+        internal static Task<IEnumerable<ContentItem>> QueryListItemsAsync(ISession session, string listContentItemId, Expression<Func<ContentItemIndex, bool>> itemPredicate = null, Expression<Func<ContainedPartIndex, object>> partPredicate = null)
```

Comment:
If there is nothing else to order by on ContainedPart I assume a `bool order` would be sufficient?

Reply:
I was thinking of adding a part for visits and analytics... So, liquid tag could be used with a custom filter...

Reply:
It's an internal method so not sure how you would do that?

Plus here you have also added a default ordering, which is necesary?

I think the razor helper `ListOrchardHelperExtensions` needs the same update